osx/scripts/postinstall-launchd-jenkins

```diff
@@ -44,7 +44,13 @@ else
     dscl . -append /Groups/jenkins GroupMembership jenkins
 fi
 
-find "$JENKINS_HOMEDIR" \( -not -user jenkins -or -not -group jenkins \) -print0 | xargs -0 chown jenkins:jenkins
+# identify the real default group name for user jenkins
+groupid=`dscl . read /Users/jenkins PrimaryGroupID | awk '{print $2}'`
+gname=`id -n -g $groupid`
+
+echo "Using jenkins:${gname} as file owner and group for jenkins daemon files"
+
+find "$JENKINS_HOMEDIR" \( -not -user jenkins -or -not -group ${gname} \) -print0 | xargs -0 chown jenkins:${gname}
```

Comment:
Yup this is the same problem as in the RPM installer. Maybe create a mac specific issue and link it to JENKINS-23273 ? It would be nice to have a similar solution to both.